main.cc

```diff
@@ -357,12 +357,39 @@ int main(int ac, char** av) {
     // If --version is requested, print it out and exit immediately to avoid
     // Seastar-specific warnings that may occur when running the app
     bpo::variables_map vm;
-    bpo::store(bpo::command_line_parser(ac, av).options(app.get_options_description()).allow_unregistered().run(), vm);
+    auto parsed_opts = bpo::command_line_parser(ac, av).options(app.get_options_description()).allow_unregistered().run();
+    bpo::store(parsed_opts, vm);
     if (vm["version"].as<bool>()) {
         fmt::print("{}\n", scylla_version());
         return 0;
     }
 
+    if (ac) {
+        fmt::print("Scylla executed using: \"{}", av[0]);
+        for (int i = 1; i < ac; ++i) {
+            fmt::print(" {}", av[i]);
+        }
+        fmt::print("\"\n");
+    }
+
+    fmt::print("Parsed command line options:\n");
+    for (auto const& opt: parsed_opts.options) {
+        if (opt.value.empty()) {
```

Comment:
Please follow the coding style: west const, spaces around `:`

Reply:
Done.